media-video/pipewire/files/99-pipewire-default-hook.conf

```diff
@@ -0,0 +1,17 @@
+# Load pipewire configuration at conf hook processing time. This allows to
+# override pulseaudio defaults configuration which is also applied via hook.
+#
+# Note since hooks are run after /etc/asound.conf and ~/.asoundrc are applied,
+# we load these again here make sure that user configuration takes precedence.
+
+@hooks [
+	{
+		func load
+		files [
+			"/usr/share/alsa/alsa.conf.d/99-pipewire-default.conf"
```

Comment:
Seeing the absolute path to /usr makes me wonder what happens in case of a Gentoo prefix installation but it might be that there such plugin doesn't even make sense making the absolute path here a red herring. Hopefully someone else will know for sure.

Reply:
`media-plugins/alsa-plugins` also uses absolute path in the installed `files/51-pulseaudio-probe.conf` so this should be fine too:
```shell
@hooks [
        {
                func pulse_load_if_running
                files [
                        "/usr/share/alsa/pulse-default.conf"
                        "/etc/asound.conf"
                        "~/.asoundrc"
                ]
                errors false
        }
]
```